sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/share_client.hpp

```diff
@@ -148,10 +148,10 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
      * @brief Gets the properties of the share.
      * @param options Optional parameters to get the share properties.
      * @param context Context for cancelling long running operations.
-     * @return Azure::Response<Models::GetSharePropertiesResult> containing the properties for
+     * @return Azure::Response<Models::ShareProperties> containing the properties for
      * ths share or one of its snapshot.
      */
-    Azure::Response<Models::GetSharePropertiesResult> GetProperties(
+    Azure::Response<Models::ShareProperties> GetProperties(
```

Comment:
need to remove `RequestId` from `ShareProperties`, because `RequestId` is not properties of a share